bitcoin/src/pow.rs

```diff
@@ -1077,21 +1077,36 @@ impl kani::Arbitrary for U256 {
     }
 }
 
+/// In test code, U256s are a pain to work with, so we just convert Rust primitives in many places
 #[cfg(test)]
-mod tests {
-    use super::*;
+pub mod test_utils {
+    use crate::pow::{Target, Work, U256};
+
+    #[allow(missing_docs)]
```

Comment:
I'd just throw a quick rustdoc comment on these "/// Converts a `u64` to a `Work`."

Reply:
will do